2022-06-17 10:01:55.724622: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
2022-06-17 10:01:55.724632: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
2022-06-17 10:01:55.724654: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
2022-06-17 10:01:55.724712: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
2022-06-17 10:01:55.724761: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
2022-06-17 10:01:55.724782: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
2022-06-17 10:01:55.724803: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
2022-06-17 10:01:55.724825: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
2022-06-17 10:01:55.724830: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
2022-06-17 10:01:55.724843: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
Traceback (most recent call last):
  File "/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/gridsearch_code/grid_3/gird_3_45.py", line 93, in <module>
    train_x = pd.read_csv("~/TCGA/Methylation450K/feature_importance_6_15/RF_gredsearch/new_RF_train_x_"+str(i)+"_"+str(m)+".csv")
  File "/public/slst/home/ningwei/anaconda3/envs/DNN/lib/python3.9/site-packages/pandas/io/parsers.py", line 610, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/public/slst/home/ningwei/anaconda3/envs/DNN/lib/python3.9/site-packages/pandas/io/parsers.py", line 462, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/public/slst/home/ningwei/anaconda3/envs/DNN/lib/python3.9/site-packages/pandas/io/parsers.py", line 819, in __init__
    self._engine = self._make_engine(self.engine)
  File "/public/slst/home/ningwei/anaconda3/envs/DNN/lib/python3.9/site-packages/pandas/io/parsers.py", line 1050, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/public/slst/home/ningwei/anaconda3/envs/DNN/lib/python3.9/site-packages/pandas/io/parsers.py", line 1867, in __init__
    self._open_handles(src, kwds)
  File "/public/slst/home/ningwei/anaconda3/envs/DNN/lib/python3.9/site-packages/pandas/io/parsers.py", line 1362, in _open_handles
    self.handles = get_handle(
  File "/public/slst/home/ningwei/anaconda3/envs/DNN/lib/python3.9/site-packages/pandas/io/common.py", line 647, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/RF_gredsearch/new_RF_train_x_3_45.csv'
2022-06-17 10:02:59.500398: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/gridview//pbs/dispatcher/lib::/usr/local/lib64:/usr/local/lib
2022-06-17 10:02:59.508335: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2022-06-17 10:02:59.509295: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (node33): /proc/driver/nvidia/version does not exist
2022-06-17 10:02:59.510346: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-06-17 10:03:03.530833: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/gridview//pbs/dispatcher/lib::/usr/local/lib64:/usr/local/lib
2022-06-17 10:03:03.530921: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2022-06-17 10:03:03.530971: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (node33): /proc/driver/nvidia/version does not exist
2022-06-17 10:03:03.531661: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-06-17 10:03:03.532649: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/gridview//pbs/dispatcher/lib::/usr/local/lib64:/usr/local/lib
2022-06-17 10:03:03.532771: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2022-06-17 10:03:03.532855: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (node33): /proc/driver/nvidia/version does not exist
2022-06-17 10:03:03.533303: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/gridview//pbs/dispatcher/lib::/usr/local/lib64:/usr/local/lib
2022-06-17 10:03:03.533385: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2022-06-17 10:03:03.533448: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (node33): /proc/driver/nvidia/version does not exist
2022-06-17 10:03:03.534079: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-06-17 10:03:03.534301: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-06-17 10:03:03.534749: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/gridview//pbs/dispatcher/lib::/usr/local/lib64:/usr/local/lib
2022-06-17 10:03:03.534817: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2022-06-17 10:03:03.534853: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (node33): /proc/driver/nvidia/version does not exist
2022-06-17 10:03:03.535590: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-06-17 10:03:03.535791: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/gridview//pbs/dispatcher/lib::/usr/local/lib64:/usr/local/lib
2022-06-17 10:03:03.535866: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2022-06-17 10:03:03.535910: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (node33): /proc/driver/nvidia/version does not exist
2022-06-17 10:03:03.536579: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-06-17 10:03:03.537581: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/gridview//pbs/dispatcher/lib::/usr/local/lib64:/usr/local/lib
2022-06-17 10:03:03.537601: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/gridview//pbs/dispatcher/lib::/usr/local/lib64:/usr/local/lib
2022-06-17 10:03:03.537652: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2022-06-17 10:03:03.537732: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (node33): /proc/driver/nvidia/version does not exist
2022-06-17 10:03:03.537733: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2022-06-17 10:03:03.537816: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (node33): /proc/driver/nvidia/version does not exist
2022-06-17 10:03:03.538362: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-06-17 10:03:03.538964: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-06-17 10:03:03.551310: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/gridview//pbs/dispatcher/lib::/usr/local/lib64:/usr/local/lib
2022-06-17 10:03:03.551391: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2022-06-17 10:03:03.551432: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (node33): /proc/driver/nvidia/version does not exist
2022-06-17 10:03:03.552244: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-06-17 10:03:03.614856: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:176] None of the MLIR Optimization Passes are enabled (registered 2)
2022-06-17 10:03:03.621957: I tensorflow/core/platform/profile_utils/cpu_utils.cc:114] CPU Frequency: 3000000000 Hz
2022-06-17 10:03:03.708102: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:176] None of the MLIR Optimization Passes are enabled (registered 2)
2022-06-17 10:03:03.708919: I tensorflow/core/platform/profile_utils/cpu_utils.cc:114] CPU Frequency: 3000000000 Hz
2022-06-17 10:03:03.718514: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:176] None of the MLIR Optimization Passes are enabled (registered 2)
2022-06-17 10:03:03.719259: I tensorflow/core/platform/profile_utils/cpu_utils.cc:114] CPU Frequency: 3000000000 Hz
2022-06-17 10:03:03.733731: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:176] None of the MLIR Optimization Passes are enabled (registered 2)
2022-06-17 10:03:03.734553: I tensorflow/core/platform/profile_utils/cpu_utils.cc:114] CPU Frequency: 3000000000 Hz
2022-06-17 10:03:03.737842: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:176] None of the MLIR Optimization Passes are enabled (registered 2)
2022-06-17 10:03:03.737975: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:176] None of the MLIR Optimization Passes are enabled (registered 2)
2022-06-17 10:03:03.738669: I tensorflow/core/platform/profile_utils/cpu_utils.cc:114] CPU Frequency: 3000000000 Hz
2022-06-17 10:03:03.738721: I tensorflow/core/platform/profile_utils/cpu_utils.cc:114] CPU Frequency: 3000000000 Hz
2022-06-17 10:03:03.753725: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:176] None of the MLIR Optimization Passes are enabled (registered 2)
2022-06-17 10:03:03.754467: I tensorflow/core/platform/profile_utils/cpu_utils.cc:114] CPU Frequency: 3000000000 Hz
2022-06-17 10:03:03.768530: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:176] None of the MLIR Optimization Passes are enabled (registered 2)
2022-06-17 10:03:03.769397: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:176] None of the MLIR Optimization Passes are enabled (registered 2)
2022-06-17 10:03:03.769516: I tensorflow/core/platform/profile_utils/cpu_utils.cc:114] CPU Frequency: 3000000000 Hz
2022-06-17 10:03:03.770648: I tensorflow/core/platform/profile_utils/cpu_utils.cc:114] CPU Frequency: 3000000000 Hz
Traceback (most recent call last):
  File "/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/gridsearch_code/grid_3/grid_3_5.py", line 171, in <module>
    f.write(grid.best_score_) #将字符串写入文件中
TypeError: write() argument must be str, not numpy.float64
Traceback (most recent call last):
  File "/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/gridsearch_code/grid_3/grid_3_10.py", line 171, in <module>
    f.write(grid.best_score_) #将字符串写入文件中
TypeError: write() argument must be str, not numpy.float64
Traceback (most recent call last):
  File "/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/gridsearch_code/grid_3/grid_3_15.py", line 149, in <module>
    f.write(grid.best_score_) #将字符串写入文件中
TypeError: write() argument must be str, not numpy.float64
Traceback (most recent call last):
  File "/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/gridsearch_code/grid_3/grid_3_20.py", line 149, in <module>
    f.write(grid.best_score_) #将字符串写入文件中
TypeError: write() argument must be str, not numpy.float64
Traceback (most recent call last):
  File "/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/gridsearch_code/grid_3/grid_3_25.py", line 149, in <module>
    f.write(grid.best_score_) #将字符串写入文件中
TypeError: write() argument must be str, not numpy.float64
Traceback (most recent call last):
  File "/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/gridsearch_code/grid_3/grid_3_30.py", line 149, in <module>
    f.write(grid.best_score_) #将字符串写入文件中
TypeError: write() argument must be str, not numpy.float64
Traceback (most recent call last):
  File "/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/gridsearch_code/grid_3/grid_3_35.py", line 149, in <module>
    f.write(grid.best_score_) #将字符串写入文件中
TypeError: write() argument must be str, not numpy.float64
Traceback (most recent call last):
  File "/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/gridsearch_code/grid_3/grid_3_40.py", line 149, in <module>
    f.write(grid.best_score_) #将字符串写入文件中
TypeError: write() argument must be str, not numpy.float64
Traceback (most recent call last):
  File "/public/slst/home/ningwei/TCGA/Methylation450K/feature_importance_6_15/gridsearch_code/grid_3/gird_3_50.py", line 149, in <module>
    f.write(grid.best_score_) #将字符串写入文件中
TypeError: write() argument must be str, not numpy.float64
